src/widgetastic/browser.py

```diff
@@ -28,6 +38,12 @@
 from .utils import crop_string_middle
 from .utils import retry_stale_element
 from .xpath import normalize_space
+from widgetastic.types import DefaultPluginType
+from widgetastic.types import ElementParent
+from widgetastic.types import LocatorAlias
+
+if TYPE_CHECKING:
+    from .widget.base import Widget
 
 Size = namedtuple("Size", ["width", "height"])
```

Comment:
i believe starting with python 3.6.1 named-tuple from typing can be used to type those as well

Reply:
Done